### DISCUSSION TITLE
タグのアイコンの履歴管理

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#60] [要望・アイデア・不具合]
各タグのアイコン画像が変な画像に変えられてしまうことがたまにあるのですが、そういった時にgit的に以前のものに戻したりができるように履歴が参照できると直すのが楽で助かります・・・！（とりあえず過去の画像が参照だけできればそれを再度アップロードして更新できるのでそういった形でも構いません）

※あると便利・・・程度なので優先度低めで構いません！

**Comment by kskwtnk** [comment]
@simon-ritchie 

ご意見ありがとうございます！

タグの画像なのですが、いただいた内容を含め管理しやすいように変えていきたい思いはあります

運営が用意するとか、ある程度活動量があるユーザーだけが変えられるようになるとか、いくらかパターンは出ているのですが
なかなかすぐに実現することは難しそう……というのが回答です！すみません！

**Reply by simon-ritchie** [reply]
@xrxoxcxox さん

> 運営が用意するとか、ある程度活動量があるユーザーだけが変えられるようになるとか、いくらかパターンは出ているのですが
> なかなかすぐに実現することは難しそう……というのが回答です！すみません！

承知いたしました！確かに活動量とかContributionのしきい値とか色々パターンがありそうですね・・・！

急ぎで必要なものでもないですし、他の優先度高めなことも色々あるでしょうからいつか対応が入ったら・・・的にゆったりとお待ちしております！
